### PR TITLE
Added Github-handle for Simone Wojtaszek

### LINKED ISSUE #6720: Add github-handle for Simone Wojtaszek in ems-triage-tracker.md
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started Page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).
   
### Overview
We need to create a single variable `github-handle` to hold the github handle for each member of the leadership team. Eventually `github-handle` will replace the `github` and `picture` variables, reducing redundancy in the project file.

### Action Items
- [x] Open the file `_projects/ems-triage-tracker.md` in the IDE
- [x] Replace:
```
- name: Simone Wojtaszek
```
with
```
- name: Simone Wojtaszek
  github-handle: 
```
- [x] Do not use a tab to indent `github-handle`. YAML doesn't allow tabs; it requires spaces.
- [x] Using docker, confirm that the appearance of the project webpage is unchanged at all screen sizes. The project webpage URL can be found below under Resources.

### For Merge Team
- [ ] Release the dependency for this issue in #5441. If all the dependencies have been completed, close that issue.

### Resources/Instructions
https://github.com/hackforla/website/wiki/project.md-file-template  
https://jekyllrb.com/  
Project Webpage: https://www.hackforla.org/projects/ems-triage-tracker

- This issue is part of #5441.

### PR DESCRIPTION
**Pull request by Limeload**
Fixes #6720 

### What changes did you make?
  - Added github-handle for Simone Wojtaszek in ems-triage-tracker.md.

### Why did you make the changes (we will use this info to test)?
  - Reducing redundancy in the project file.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
  - Added github-handle to the file. No visual changes to the website.

### CODE REVIEW

**Comment by jchue** [comment]
Review ETA: 8 PM 5/22/24
Availability: 5/22/24 5-8 PM PDT

File: _projects/ems-triage-tracker.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ leadership:
       github: 'https://github.com/McRawly'
     picture: https://avatars.githubusercontent.com/McRawly
   - name: Simone Wojtaszek
+  github-handle:
```
Comment (jchue) [inline _projects/ems-triage-tracker.md]: Hi @Limeload , thanks for taking this on!

I think the indentation here is a little off; it should be indented two more spaces to align with `role` underneath.

**Comment by vanessasinam** [comment]
Review ETA: 8 PM 5/22/24
Availability: 5/22/24 4-8 PM EST

**Comment by del9ra** [comment]
Review ETA: 10 PM 5/23/24
Availability: 1-5 PM Friday

**Review by del9ra — APPROVED** [review]
Hi @Limeload,
* The pull request is done with the correct branch
* The issue is properly linked
* Why did you modify other lines of code? You were supposed to add only one “github-handle:”
* I struggled to find your fork at “https://github.com/Limeload/hackforlawebsite”. Isn’t it supposed to be called “https://github.com/Limeload/website”?
* The PR title is descriptive of the changes
* Your explanation of changes is clear 
* The look of the project webpage hasn't changed
Great job! Thanks for taking the time to contribute to the website!

**Review by jchue — APPROVED** [review]
@Limeload Thanks for making the corrections!

- Pull request done with correct branch
- Correct issue linked
- Indents made with spaces
- Project webpage unchanged

Looks good now!

**Pull request merged**